runtime/src/main/java/com/datastax/oss/quarkus/runtime/graal/NativeSubstitutions.java

```diff
@@ -15,20 +15,40 @@
  */
```

Comment:
Changes to this class avoid the observed issues with ClassLoader.defineClass() via jnr by replacing the loading logic in the base impl of Native with this no-op version.

Note that this is only a temporary fix.  The _right_ fix for this problem will be provided by JAVA-2663, but until we get a driver version including that change we'll need something like this to remove jnr from consideration in the Graal case.

Reply:
Wouldn't it be simpler to just use driver 4.7.0-SNAPSHOT for now? But otherwise I'm fine with the changes here as long as we don't forget to revert them when 2663 is available.

Reply:
We'll have to make _some_ change when 2663 comes in anyway; once we get to that spot the Native substitutions should come out entirely.  So since we already have to do work for that situation this seemed easier than publishing a snapshot.